src/pages/index.js

```diff
@@ -54,9 +55,35 @@ const IndexPage = (props) => {
                         </Box>
                     </Box>
                 </Link>
-            )
+            );
         }
-    })
+    });
+
+    const [thankYou, setThankYou] = React.useState([]);
+
+    useEffect(() => {
+        axios
+            .get(
+                'https://raw.githubusercontent.com/jenkins-infra/jenkins-contribution-stats/main/data/honored_contributor.csv',
+                { responseType: 'text' }
+            )
+            .then((response) => {
+                setThankYou(Papa.parse(response.data)?.data[1]);
+            });
+
+        const interval = setInterval(() => {
+            axios
+                .get(
+                    'https://raw.githubusercontent.com/jenkins-infra/jenkins-contribution-stats/main/data/honored_contributor.csv',
+                    { responseType: 'text' }
+                )
+                .then((response) => {
+                    setThankYou(Papa.parse(response.data)?.data[1]);
+                });
+        }, 3600000);
```

Comment:
Non blocking: can you explain a bit why this is set to 1000 hours?

Reply:
It is 1 hour or 3600 seconds

Reply:
The units used are milliseconds.